core/testcontainers/core/container.py

```diff
@@ -155,3 +159,54 @@ def exec(self, command) -> tuple[int, str]:
         if not self._container:
             raise ContainerStartException("Container should be started before executing a command")
         return self._container.exec_run(command)
+
+
+class Reaper:
+    _instance: Optional[Reaper] = None
```

Comment:
presumably if this is here we can actually reuse ryuk on multiple instances but then the actual code for telling ryuk about it is not in a regular instance public method - so not sure but i think this part could be more coherent